test/conversations/Conversations.test.ts

```diff
@@ -368,4 +370,24 @@ describe('conversations', () => {
       expect(invites).toHaveLength(1)
     })
   })
+
+  describe('newGroupConversation', () => {
+    it('sends invites to recipients', async () => {
+      const context = {
+        conversationId: '',
+        metadata: {
+          groupID: 'the-group-id',
```

Comment:
What if we moved this to the `conversationId` field. Something like `xmtp.org/group/$GROUP_ID`.

Reply:
I guess the question is what if folks want to use conversationIds with their group conversations?

Reply:
I'm ok with removing that possibility

Reply:
It sort of feels weird to me that we offer `conversationId` to developers for their own use except for in this case where we reserve it. What about a new property of Invitation context?

Reply:
We don't have the same deduping rules for group convos that we do for 1:1. 

ConversationId is useful in 1:1 because if you don't supply one, you might get an existing convo returned from `newConversation`. When creating a new group convo, you will always get a brand new convo (even with the same members). 

So, I don't think developers are missing much. 

We could let them set metadata optionally for groups, and merge it with our metadata.